exporters/otlp/otlpmetric/otlpmetricgrpc/internal/oconf/envconfig_test.go

```diff
@@ -113,28 +113,50 @@ func TestWithEnvAggPreference(t *testing.T) {
 			name:     "explicit_bucket_histogram",
 			envValue: "explicit_bucket_histogram",
 			want: map[metric.InstrumentKind]metric.Aggregation{
-				metric.InstrumentKindCounter:                 metric.DefaultAggregationSelector(metric.InstrumentKindCounter),
-				metric.InstrumentKindHistogram:               metric.DefaultAggregationSelector(metric.InstrumentKindHistogram),
-				metric.InstrumentKindUpDownCounter:           metric.DefaultAggregationSelector(metric.InstrumentKindUpDownCounter),
-				metric.InstrumentKindObservableCounter:       metric.DefaultAggregationSelector(metric.InstrumentKindObservableCounter),
-				metric.InstrumentKindObservableUpDownCounter: metric.DefaultAggregationSelector(metric.InstrumentKindObservableUpDownCounter),
-				metric.InstrumentKindObservableGauge:         metric.DefaultAggregationSelector(metric.InstrumentKindObservableGauge),
```

Comment:
I'm not a fan of this particular case, but I'm good with the overall changes.

Reply:
If you have smaller improvements to the formatting that will conform with the used formatter I'm happy to include them here. If they are larger, I'm happy to review any follow-up PRs you would like to submit to change the layout.

Reply:
120 sounds reasonable to me especially that even Effective Go kind of says that long lines can be fine.

I sometimes use "wrap line" editor feature if the lines are long and I do not want to scroll horizontally.